platform/app/public/config/default.js

```diff
@@ -12,6 +12,7 @@ window.config = {
   showCPUFallbackMessage: true,
   showLoadingIndicator: true,
   strictZSpacingForVolumeViewport: true,
+  autoPlayCine: true,
```

Comment:
let's leave this out of here and just document it

Reply:
Ah good catch. Yes I did not mean to commit this. Thanks.